content/blog/wingingit/winging-it-01.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 1. CSS Cascade Layers
+author:
+  - miriam
+  - ed
+  - stacy
+date: 2023-07-09
+tags:
+  - CSS
+  - Cascade Layers
+length: 40 mins
+action:
+  text: Watch on YouTube »
+  url: https://youtu.be/FwICaSE8iuY
+image:
+  src: winging-it/winging-it-1.jpg
+summary: |
+  If you've ever found yourself in a specificity war, you'll understand how
+  important having control over style priority can be. During our conversation,
+  we discussed what CSS Layers are and how you can use them in your project.
+---
+
+## What we cover:
```

Comment:
Do we want time codes? Related: should we break the video into chapters, or nah?

Reply:
eventually, yes. I would like to discuss how much time we'd want to allow for this as I am worried the amount of time I've spent getting this up and running is more than I think I expected at least. I think the first Episode is going to take way longer than things moving forward as we now have a lot of the base assets we need for each episode moving forward.